package.json

```diff
@@ -5,8 +5,9 @@
     "test": "npx lerna exec npm t",
     "build": "npx lerna run build",
     "docs": "npx typedoc",
-    "publish:release-candidate": "npx lerna publish --allow-branch release-candidate --dist-tag next --preid rc --no-verify-access --conventional-prerelease",
-    "publish:latest": "npx lerna publish --allow-branch main --create-release github --no-verify-access --conventional-graduate",
+    "publish:release-candidate": "npx lerna publish --allow-branch release-candidate --dist-tag next --preid rc --no-verify-access --conventional-prerelease bump preminor",
+    "publish:release": "npx lerna publish --allow-branch main --dist-tag latest --no-verify-access --conventional-graduate bump minor",
```

Comment:
was it intentional to drop creating the release in github?

Reply:
I did intentionally remove this because it's a PITA to fix tags if something gets screwed up. We have to go in and delete all 5000 tags from GitHub if we need to redo something. This shouldn't be necessary going forward, hopefully, but it might till any bugs get ironed out.

This will also stop releases from being available on GitHub going forward though. Do we care that much about them? Don't think people download packages from GitHub. I can add this back and hope we don't have to fix tags lol.

Reply:
Actually, you know what, scratch that. The git tags will get pushed anyway... I'll just add this back.